### DISCUSSION TITLE
Can't import avatar recommended asset packs

### DISCUSSION THREAD

**Discussion by akihiro-shibue** [mozilla/hubs#3802] [Hubs Cloud Questions] [answered]
Hi, there

I just started using hubs and i try to import avatars from hubs.mozilla.com but it doesn't work. Any idea what I need to fix this?
uploading won't finish and can't get any avatars and scenes.  

What I did is 

1, I got URL from below page
![image](https://user-images.githubusercontent.com/77717610/106227467-91da0680-622c-11eb-9318-67d1a0044909.png)

2,copy and paste
![image](https://user-images.githubusercontent.com/77717610/106226946-b08bcd80-622b-11eb-9fcb-8fa9b68c60e5.png)

3,ring cursor won't finish,,,,,

Thank you!

**Comment by robinkwilson** [comment]
Hi! 

1. Are you using a custom client?
2. What instance size are you using?
3. Can you open your browser's developer tools and send me some screenshots of any red errors?

**Reply by akihiro-shibue** [reply]
Hi!

1, No, I'm not. I am trying to import default avatars. 
2.I am using t3.micro
3.what kinds of screenshot can I send you? AWS? Sorry I am beginner....

Thank you!

**Comment by robinkwilson** [comment, marked as answer]
Ah yes. The t3.micro has memory issues so it will likely fail with the avatar import. 

Can you scale up to a t3.small and see if the import completes? Thank you!

**Reply by robinkwilson** [reply]
Marking as answered for now, please unmark as answered if you need more help!
